services/dynamic-scheduler/tests/unit/api_frontend/conftest.py

```diff
@@ -92,13 +93,16 @@ async def _run_server() -> None:
 
     server_task = asyncio.create_task(_run_server())
 
+    home_page_url = (
+        f"http://{server_host_port}{get_settings().DYNAMIC_SCHEDULER_UI_MOUNT_PATH}"
```

Comment:
same here. Actually now pydantic comes with a handy URL object. Probably you have another with yarl and yet another with httpx

Reply:
please see my reply above